src/BloomExe/Book/AppearanceSettings.cs

```diff
@@ -707,6 +713,26 @@ public string GetCssOwnPropsDeclaration(dynamic properties, AppearanceSettings p
             ? (System.Collections.Generic.List<object>)props[kOverrideGroupsArrayKey]
             : null;
 
+        if (props.ContainsKey(kPageNumberPositionVar))
+        {
+            SetPageNumberProperties(props[kPageNumberPositionVar].ToString());
+        }
+
+        // pageNumber-show may be set by SetPageNumberProperties, or it may get set directly in a legacy situation.
+        // This is a special case (for now) where we emit a special property designed to be used when setting the bottom margin.
+        // Search our less/css for --pageNumber-show-multiplicand to see how it is used.
+        // If we end up with other boolean properties that the user can set a more general solution will probably be needed.
```

Comment:
Not sure what to do with this comment block. Do we still want to keep these thoughts?